mod/state-transition/pkg/core/core_test.go

```
@@ -208,6 +175,108 @@ func initTestStores() (
+func progressStateToSlot(
+	t *testing.T,
+	beaconState *TestBeaconStateT,
+	slot math.U64,
+) {
+	t.Helper()
+
+	if slot == math.U64(0) {
+		t.Fatal("for genesis slot, use InitializePreminedBeaconStateFromEth1")
+	}
+
+	beaconState.SetSlot(slot)
```

golangci-lint (1.62.2)

Error return value of `beaconState.SetSlot` is not checked

(errcheck)